projects/core/src/lib/users/quota/quota-info/quota-info.component.html

```diff
@@ -0,0 +1,23 @@
+<ng-container *ngIf="hasQuota()">
+    <span class="pair">
+        <span class="label">Available credits</span>
+        <span class="value">{{ sessionQuota?.available }} of {{ sessionQuota?.total }}</span>
+    </span>
+    <span class="pair">
+        <span class="label">Used credits (total)</span>
+        <span class="value">{{ sessionQuota?.used }} of {{ sessionQuota?.total }} </span>
+    </span>
+    <span class="pair">
+        <span class="label">Used credits (percent)</span>
+        <span class="value">{{ sessionQuota?.percentUsed }}%</span>
+        <mat-progress-bar mode="determinate" [value]="sessionQuota?.percentUsed" [color]="progressColor"></mat-progress-bar>
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/218266154-3b8b6c0e-6757-403b-b486-d5c4d92d918b.png)

This should be 8%.

Reply:
[9cbccf5](https://github.com/geo-engine/geoengine-ui/pull/586/commits/9cbccf56c567b103e556ba219e6260ed50dd9b63)

Reply:
i removed all the percents
[e46564b](https://github.com/geo-engine/geoengine-ui/pull/586/commits/e46564b203836f6ce2baf1a8d4eec605fc520333)